core/dbt/parser/unit_tests.py

```diff
@@ -203,6 +206,22 @@ def __init__(self, schema_parser: SchemaParser, yaml: YamlBlock) -> None:
         self.schema_parser = schema_parser
         self.yaml = yaml
 
+    def load_rows_from_seed(self, seed_name):
+        rows = []
+
+        try:
+            seed_node = self.manifest.ref_lookup.perform_lookup(
+                f"seed.{self.project.project_name}.{seed_name}", self.manifest
```

Comment:
manifest.ref_lookup.find can be used to avoid having to reconstruct the seed unique id like this. It returns None instead of raises an error, so the handling downstream would change a bit.